packages/components/src/dialog/src/Dialog.css

```diff
@@ -17,15 +17,27 @@
     align-items: flex-start;
 }
 
+.o-ui-dialog-wrapper-fullscreen {
+    padding: 0;
+}
```

Comment:
Does a dialog still support fullscreen?

Reply:
> Doesn't work for NumberInput

Dashlane should not autofill a number input though :D

Reply:
@fraincs yes for "age" it would :)